examples/app.py

```diff
@@ -35,35 +51,93 @@
 
 You can find the database in `examples/app.db`.
 
-Run the development server:
+
+3. Transform the file `authority.xml` from MARC21 to a json file:
 
 .. code-block:: console
 
-    $ flask -a app.py run -p 5000 -h '0.0.0.0'
+    $ dojson do -i authority.xml -l marcxml marc21 > records.json
+
+
+4. Load the file in the application importing the json file:
+
+.. code-block:: console
+
+    $ flask -a app.py records create records.json
+
+
+5. Create a root collection (By default, the first collection is the root
+   node) using the CLI tool:
+
+.. code-block:: console
+
+    $ flask -a app.py collections create RootCollection
+
+
+6. Create a sub-collection with a query to show only the collections of
+   'primary' INSTITUTE:
+
+.. code-block:: console
+
+    $ flask -a app.py collections create Institutes \
+      -q 'collections.primary:INSTITUTE'
+
+
+7. Create a sub-collection with a query to show only the nuclear research
+   center and its parent is `Institutes`:
+
+.. code-block:: console
+
+    $ flask -a app.py collections create Experimentelle -p Institutes \
+      -q 'citation_references_note.name_of_source:Experimentelle'
+
+
+8. Launch the web server:
+
+.. code-block:: console
+
+    $ flask -a app.py run
+
+
+9. Open in a browser the page `http://127.0.0.1:5000/COLLECTION/`. where
+`COLLECTION` is the name of some collection previously created
+(`RootCollection`, `Institutes`, `Institutes`)
+
 """
 
 from __future__ import absolute_import, print_function
 
-import os
-
-from flask import Flask
-from flask_babelex import Babel
-from flask_breadcrumbs import Breadcrumbs
+from flask import Flask, jsonify, request
 from flask_cli import FlaskCLI
-from flask_menu import Menu
 from invenio_db import InvenioDB
+from invenio_records import InvenioRecords
+from invenio_search import InvenioSearch, Query, current_search_client
 
 from invenio_collections import InvenioCollections
+from invenio_collections.models import Collection
 
 # Create Flask application
 app = Flask(__name__)
 app.config.update(
-    SQLALCHEMY_DATABASE_URI=os.getenv('SQLALCHEMY_DATABASE_URI',
-                                      'sqlite:///app.db'),
+    SEARCH_ELASTIC_KEYWORD_MAPPING={None: ['_all']},
+    SQLALCHEMY_DATABASE_URI='sqlite:///app.db',
 )
+
 FlaskCLI(app)
-Babel(app)
-Menu(app)
-Breadcrumbs(app)
 InvenioDB(app)
+InvenioRecords(app)
 InvenioCollections(app)
+InvenioSearch(app)
+
+
+@app.route('/<path:collection_name>/', methods=['GET'])
+def index(collection_name):
+    """Query Elasticsearch using a collection"""
+    collection = Collection.query.filter_by(name=collection_name).one()
+    query = Query(collection.dbquery)
```

Comment:
@jirikuncar What do you mean? I know there is a field with the `_collections` but my idea was to search with InvenioSearch a defined collection. Do you prefer to remove the InvenioSearch dependency and just iterate over all the records checking if they are in the specified collection?